README.md

```diff
@@ -10,7 +10,7 @@ and [git](https://github.com/Kottans/git-course) courses.
 You can try sending pull-requests to this repository proposing a change.
 The change can be anything you like, for example: fixing a typo in README.md. Consider this phrase, for example:
 
-*This was an intentionaly mispelld phrase*
+*This was an intentionaly mispelled phrase*
```

Comment:
Could you please change this word to previous? (Most of the students do not have these changes, and this will cause a conflict in their files)